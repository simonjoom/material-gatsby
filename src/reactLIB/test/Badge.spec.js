import React from 'react';
import { shallow } from 'enzyme';
import Badge from '../src/Badge';

describe('<Badge />', () => {
  let wrapper;

  test('renders', () => {
    wrapper = shallow(<Badge>4</Badge>);
    expect(wrapper).toMatchSnapshot();
  });

  test('should output a badge with the className of `badge`', () => {
    wrapper = shallow(<Badge>4</Badge>);
    expect(wrapper).toMatchSnapshot();
  });

  test('should output a badge with the className of `badge new` and', () => {
    wrapper = shallow(<Badge newIcon>3</Badge>);
    expect(wrapper).toMatchSnapshot();
  });
});
