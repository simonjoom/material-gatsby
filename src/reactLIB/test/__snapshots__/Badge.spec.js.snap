// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Badge /> renders 1`] = `
<span
  className="badge"
>
  4
</span>
`;

exports[`<Badge /> should output a badge with the className of \`badge new\` and 1`] = `
<span
  className="badge new"
>
  3
</span>
`;

exports[`<Badge /> should output a badge with the className of \`badge\` 1`] = `
<span
  className="badge"
>
  4
</span>
`;
